### ISSUE TITLE
vmagent metrics per (scrape) job

### ISSUE THREAD

**Issue by Lusitaniae** [VictoriaMetrics/VictoriaMetrics#6169]
### Is your feature request related to a problem? Please describe

I'm using vmagent to scrape 30 different "jobs" across 100s of nodes.

It's difficult to tell which one is the most resource hungry (network/CPU) without per job metrics 

### Describe the solution you'd like

Ideally we'd have vmagent metrics per job such as:

```
scrape_target_up{job=a} 10
scrape_sample_rate{job=a} 20
metrics_scraped{job=a} 100
scrape_response_time{job=a}
scrape_response_size{job=a}
```

Could be `scrape_job=a` to not conflict with `metric{job=vmagent}`

Also some extra stuff like metrics/labels dropped by vmagent

Basically all the metrics on this dashboard could have a per job label for further filtering
https://grafana.com/grafana/dashboards/12683-victoriametrics-vmagent/

**Comment by mkilchhofer** [comment]
This would be a nice enhancement of:
- https://github.com/VictoriaMetrics/VictoriaMetrics/issues/5311

The enhancement of #5311 makes it possible to isolate a failing scrape target to the scrape job within the scrape config. Since we mostly use the default config of the helm chart, most workloads are scraped via `job_name: "kubernetes-service-endpoints"` and thus it is only possible to evaluate the job_name:

> vm_promscrape_scrape_pool_targets{cluster="k3s", instance="localhost:8429", job="vmagent", namespace="monitoring", pod="vmagent-victoria-metrics-agent-0", scrape_job="kubernetes-service-endpoints", **status="up"**} 22
> vm_promscrape_scrape_pool_targets{cluster="k3s", instance="localhost:8429", job="vmagent", namespace="monitoring", pod="vmagent-victoria-metrics-agent-0", **scrape_job="kubernetes-service-endpoints"**, **status="down"**} 1

I then always need to use the VMangent webui to find the real failing job
![grafik](https://github.com/user-attachments/assets/94638951-45b6-476a-ab51-ed885b8b911c)
